### DISCUSSION TITLE
How to prevent the same file to be added in the playlist multiple times?

### DISCUSSION THREAD

**Discussion by johnmapeson** [losnoco/Cog#401] [Q&A]
And the second question:

If I click `test1` or `test2` in the sidebar multiple times, it will be added to the playlist again and again. How to prevent this?

<img width="384" alt="image" src="https://github.com/user-attachments/assets/a72220a9-69cc-4851-ae5a-8fa5259bf9ae">

**Comment by kode54** [comment]
Unless you're proposing that clicking the sidebar always replaces the entire playlist, I'd have to run a duplicate track check on the playlist every time it's clicked on. And which duplicates should be removed? What if a user wants duplicates?
